src/openrct2/drawing/ImageImporter.cpp

```diff
@@ -362,15 +362,15 @@ const PaletteBGRA ImageImporter::StandardPalette[256] =
     { 0, 0, 0, 255 },
 
     // 1 - 9 (misc. e.g. font and water)
-    { 0, 0, 0, 255 },
-    { 0, 0, 0, 255 },
-    { 0, 0, 0, 255 },
-    { 0, 0, 0, 255 },
-    { 0, 0, 0, 255 },
-    { 0, 0, 0, 255 },
-    { 0, 0, 0, 255 },
-    { 0, 0, 0, 255 },
-    { 0, 0, 0, 255 },
+    { 0, 0, 1, 255 },
```

Comment:
Why did you change these? The third octet represents red, not an index, right?

Reply:
Mostly to help distinguish the separate indices a bit. I can revert this bit if preferred.

Reply:
Why didn't you just use comments?

Reply:
I wanted it to help distinguish colours in image modifying programs. Comments don't help there.

Again, I can revert this bit.

Reply:
Oh I see. In that case shouldn't you do increments of 20 or perhaps different hues?